packages/legend-graph/src/graphManager/action/changeDetection/StoreRelational_ObserverHelper.ts

```diff
@@ -998,12 +1003,61 @@ export const observe_AuthenticationStrategy = (
   return metamodel;
 };
 
+const observe_Abstract_Mapper = (metamodel: Mapper): void => {
+  makeObservable(metamodel, {
+    from: observable,
+    to: observable,
+    hashCode: computed,
+  });
+};
+
+export const observe_SchemaNameMapper = skipObserved(
+  (metamodel: SchemaNameMapper): SchemaNameMapper => {
+    observe_Abstract_Mapper(metamodel);
+
+    return metamodel;
+  },
+);
+
+export const observe_TableNameMapper = skipObserved(
+  (metamodel: TableNameMapper): TableNameMapper => {
+    observe_Abstract_Mapper(metamodel);
+
+    makeObservable(metamodel, {
+      schema: observable,
+    });
+    observe_SchemaNameMapper(metamodel.schema);
+    return metamodel;
+  },
+);
+
+export const observe_Mapper = (metamodel: Mapper): Mapper => {
+  if (metamodel instanceof SchemaNameMapper) {
+    return observe_SchemaNameMapper(metamodel);
+  } else if (metamodel instanceof TableNameMapper) {
+    return observe_TableNameMapper(metamodel);
+  }
+  return metamodel;
+};
+
+const observe_Abstract_PostProcessor = (metamodel: PostProcessor): void => {
+  makeObservable(metamodel, {
+    hashCode: computed,
+  });
+};
+
 export const observe_PostProcessor = (
   metamodel: PostProcessor,
   context: ObserverContext,
 ): PostProcessor => {
+  observe_Abstract_PostProcessor(metamodel);
```

Comment:
let's create a separate method `observe_MapperPostProcessor`, just mimic what you already did for `observe_Mapper` above. If you put `observe_Abstract_PostProcessor()` like this, the instance is already observed, if so, we run the risk of `observe_XPostProcessor()` being skipped if people use `skipObserved()`

Reply:
Let's put this one on line `1052` as explained. Any `post-processor observers` will need to call this

Reply:
did you forget about this?